apps/pdl/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/pdl/HentPersonerRiver.kt

```diff
@@ -109,12 +109,11 @@ class HentPersonerRiver(
         )
 
     private fun hentPersoner(fnrListe: Set<Fnr>): List<Person> =
-        Metrics.pdlRequest
-            .recordTime(pdlClient::personBolk) {
-                pdlClient.personBolk(
-                    fnrListe.map(Fnr::verdi),
-                )
-            }.orEmpty()
+        runBlocking {
+            pdlClient.personBolk(
+                fnrListe.map(Fnr::verdi),
+            )
+        }.orEmpty()
```

Comment:
Måtte gjøre en endring for å trigge deploy av PDL. Gjorde derfor en liten opprydding her.

Reply:
Ja, vi må / bør vel evt sette opp noen nye metrikker på sikt